google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -426,24 +519,11 @@ public static EnhancedBigtableStub create(EnhancedBigtableStubSettings settings)
             clientContext.getTracerFactory(),
             SpanName.of(TRACING_OUTER_CLIENT_NAME, methodName));
 
-    return traced.withDefaultCallContext(clientContext.getDefaultCallContext());
-  }
-
-  /**
-   * Wraps a callable chain in a user presentable callable that will inject the default call context
-   * and trace the call.
-   */
-  private <RequestT, ResponseT>
-      ServerStreamingCallable<RequestT, ResponseT> createUserFacingServerStreamingCallable(
-          String methodName, ServerStreamingCallable<RequestT, ResponseT> inner) {
-
-    ServerStreamingCallable<RequestT, ResponseT> traced =
-        new TracedServerStreamingCallable<>(
-            inner,
-            clientContext.getTracerFactory(),
-            SpanName.of(TRACING_OUTER_CLIENT_NAME, methodName));
+    UnaryCallable<RequestT, ResponseT> measured =
+        new MeasuredUnaryCallable<>(
+            traced, methodName, tagger, statsRecorder, clientContext.getClock());
```

Comment:
Should the methodName parameter include `TRACING_OUTER_CLIENT_NAME`? considering this as user-facing.